### ISSUE TITLE
Dr. Soltoff Interaction Terms Code

### ISSUE THREAD

**Issue by johnhenrypezzuto** [UC-MACSS/persp-research_Spr18#20]
Where can I find the code Dr. Soltoff used in his interaction term slides?

**Comment by yilundai** [comment]
The code is in R. Is there any python code?

**Comment by bensoltoff** [comment]
No. But it isn't difficult to write it in Python. Use `statsmodel` and the OLS function to estimate the model, then [extract the variance-covariance matrix](http://www.statsmodels.org/dev/generated/statsmodels.regression.linear_model.RegressionResults.cov_params.html#statsmodels.regression.linear_model.RegressionResults.cov_params). From there you can manually calculate the standard errors of the quasi-coefficients and generate similar plots to the notes.
